ext/AbstractFFTsForwardDiffExt.jl

```diff
@@ -0,0 +1,66 @@
+module AbstractFFTsForwardDiffExt
+
+using AbstractFFTs
+using AbstractFFTs.LinearAlgebra
+import ForwardDiff
+import ForwardDiff: Dual
+import AbstractFFTs: Plan, mul!, dualplan, dual2array
+
+
+AbstractFFTs.complexfloat(x::AbstractArray{<:Dual}) = AbstractFFTs.complexfloat.(x)
+AbstractFFTs.complexfloat(d::Dual{T,V,N}) where {T,V,N} = convert(Dual{T,float(V),N}, d) + 0im
+
+AbstractFFTs.realfloat(x::AbstractArray{<:Dual}) = AbstractFFTs.realfloat.(x)
+AbstractFFTs.realfloat(d::Dual{T,V,N}) where {T,V,N} = convert(Dual{T,float(V),N}, d)
```

Comment:
Wouldn't it be sufficient to define `fftfloat` or `_fftfloat` (not sure ad hoc which one would be preferrable)?

Reply:
Yes I think you are right, I'll try changing this